ci/unit/test_post_always.sh

```diff
@@ -29,6 +29,18 @@ ssh "$SSH_KEY_ARGS" jenkins@"$NODE" \
    DAOS_BASE=$DAOS_BASE             \
    $(cat "$mydir/test_post_always_node.sh")"
 
+use_rsync=0
```

Comment:
Use bash's booleans:

```suggestion
use_rsync=false
```